docs/docusaurus.config.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
           position: 'left',
           label: 'Docs',
         },
-        { to: '/blog', label: 'Blog', position: 'left' },
```

Comment:
Blog was not used and `/blog` did not exist so caused errors.  Removed blog to fix.